test/ngResource/resourceSpec.js

```diff
@@ -1978,4 +1978,39 @@ describe('configuring `cancellable` on the provider', function() {
     expect(creditCard3.$cancelRequest).toBeDefined();
   });
 });
+
+  describe('exception handling in callbacks', function() {
```

Comment:
These tests should be incorporated into the [errors block][1]. Also, can you add tests for all 4 combinations (no errorCallback/no responseErrorInterceptor, no errorCallback/yes responseErrorInterceptor, yes errorCallback/no responseErrorInterceptor, yes errorCallback/yes responseErrorInterceptor).

[1]: https://github.com/kylewuolle/angular.js/blob/982a063507f649bde84b727dcc2cefcf3e0bbcd3/test/ngResource/resourceSpec.js#L1618